app/scripts/modules/core/src/presentation/details.less

```diff
@@ -66,7 +66,7 @@
     flex-flow: row wrap;
 
     margin: 0 -4px;
-    .dropdown {
+    .btn-sm {
```

Comment:
All buttons in the styleguide should match the recommended size. In case, you want to use them later <button className="primary"> or <button className="passive">